src/ResourceManager/Kusto/Commands.Kusto/Commands/RemoveAzureRmKustoCluster.cs

```diff
@@ -0,0 +1,97 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Management.Automation;
+using System.Security.Permissions;
+using Microsoft.Azure.Commands.Kusto.Models;
+using Microsoft.Azure.Commands.Kusto.Properties;
+using Microsoft.Azure.Commands.Kusto.Utilities;
+
+namespace Microsoft.Azure.Commands.Kusto
+{
+    [Cmdlet(VerbsCommon.Remove, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "KustoCluster", SupportsShouldProcess = true, DefaultParameterSetName = CmdletParametersSet),
+        OutputType(typeof(PSKustoCluster))]
```

Comment:
Doesn't look like you actually print the deleted cluster.  Please change this to bool and add a PassThru parameter: 
https://github.com/Azure/azure-powershell/blob/master/src/ResourceManager/Maps/Commands.Maps/MapsAccount/RemoveAzureMapsAccount.cs#L70
https://github.com/Azure/azure-powershell/blob/master/src/ResourceManager/Maps/Commands.Maps/MapsAccount/RemoveAzureMapsAccount.cs#L107